core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -24,6 +24,29 @@ var ExplorationEditorTranslationTab = function() {
   var translationWelcomeModal = element(
     by.css('.protractor-test-translation-tab-welcome-modal'));
 
+  this.saveUploadedAudioButton = element(
+    by.className('protractor-test-save-button'));
+
+  this.errorMessage = element(by.css('div.error-message'));
+
+  this.colorOfFirstState = function() {
+    return (element.all(by.css(
+      'rect.protractor-test-node-background.init-node')).last().
+      getCssValue('fill'));
+  };
+
+  this.colorOfNormalNode = function() {
+    return (element.all(by.css(
+      'rect.protractor-test-node-background.normal-node')).last().
+      getCssValue('fill'));
+  };
+
+  this.colorOfThirdState = function() {
```

Comment:
Name this as per the class you are extraction (I think that is a better naming to use). Third may not correspond to terminal-node in all cases I think?

Reply:
Yes, making the change. :)